SimCalorimetry/EcalEBTrigPrimProducers/plugins/EcalEBTrigPrimPhase2ESProducer.h

```diff
@@ -0,0 +1,83 @@
+#ifndef SimCalorimetry_EcalEBTrigPrimProducers_EcalEBTrigPrimPhase2ESProducer_H
+#define SimCalorimetry_EcalEBTrigPrimProducers_EcalEBTrigPrimPhase2ESProducer_H
+
+// user include files
+#include "FWCore/Framework/interface/ESProducer.h"
+#include "FWCore/Framework/interface/ModuleFactory.h"
+
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/FileInPath.h"
+
+#include "CondFormats/DataRecord/interface/EcalTPGCrystalStatusRcd.h"
+#include "CondFormats/DataRecord/interface/EcalTPGPhysicsConstRcd.h"
+
+//#include "CondFormats/DataRecord/interface/EcalTPGSpikeRcd.h"
```

Comment:
Please remove commented out include files here and everywhere else.

Reply:
as in other cases I will not remove the commented lines. I have added, as in other cases, a comment